eng/Publishing.props

```diff
@@ -87,7 +87,8 @@
   </ItemGroup>
 
   <Target Name="PublishSdkAssetsAndChecksums"
-          BeforeTargets="Publish">
+          BeforeTargets="Publish"
+          Condition="$(DotNetPublishUsingPipelines)">
```

Comment:
Should this not be like:

```
'$(DotNetPublishUsingPipelines)' == 'true'
```
?

Reply:
Same diff: https://github.com/dotnet/arcade/blob/f9ce6058792211e652243db3b07fa511cf83486b/src/Microsoft.DotNet.Arcade.Sdk/tools/Publish.proj#L45

Reply:
I'd argue both should be corrected but I see in most places [it's explicitly defaulted](https://github.com/dotnet/arcade/blob/a4191734058802ac065b0f3270b76d2f0f3a60cd/src/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj#L89)